cloudbuild-pr.yaml

```diff
@@ -81,6 +81,17 @@ steps:
       ../gcs-service-account.json
       $PROJECT_ID
 
+- name: 'gcr.io/cloud-builders/kubectl'
+  id: run-migrations
+  waitFor: ['helm-deploy-studio-instance']
```

Comment:
why do we run the migrations after deploying the code? I'm sorry I don't quite understand here. I thought we want to run the migrations before deploying the code.

Reply:
Agree, this isn't the ideal location. Unfortunately I can't run the migration inside the helm deploy. Helm has a timeout, and it waits for any jobs it runs to complete. So any jobs that we create from helm need to be run outside of it.

Upon more pondering, we can create the job outside of helm *before*. Let me make make that change!